backend/routes/api.php

```diff
@@ -24,7 +25,7 @@
         return UserResource::make($request->user());
     });
     Route::apiResource('users', UserController::class)->only(['index']);
-    Route::get('/users/{user_id}/accounts', [UserController::class, 'myAccounts']);
+    Route::resource('users.accounts',UserController::class)->shallow();
```

Comment:
also, its using UserController? why?

the nested resource is Accounts, i expect it to be Account

![image](https://github.com/framgia/sph-flutter/assets/95029803/72c7aeed-7f96-4119-a452-2e82c38553f1)